### PR TITLE
Ava/checklist-ify

### PR DESCRIPTION
**Pull request by avahoffman**
Playing around with some rewording/formatting

### CODE REVIEW

**Comment by avahoffman** [comment]
@carriewright11 What do you think of this? Mostly the same, just rearranged with emoji 😁

File: 04-writing_a_DMS_plan.Rmd [hunk 1]
```diff
@@ -23,42 +18,66 @@ These following are the major sections of the plan:
...
+
+### Metadata, other relevant data, and associated documentation
+
+☑️  List information about metadata.
+&nbsp;
 
-B. Data Sharing
+☑️  List other possibly relevant data.  
 
-- Describe what data from the project will shared or not shared and justify this decision.
```
Comment (carriewright11) [inline 04-writing_a_DMS_plan.Rmd:37]: Might still want this right?

File: 04-writing_a_DMS_plan.Rmd [hunk 2]
```diff
@@ -70,72 +89,72 @@ Standards vary by discipline. An extensive list of standards you can search can
...
+&nbsp;
 
+☑️  Describe any restrictions imposed by federal, Tribal, or state laws, regulations, or policies, or existing or anticipated agreements
+&nbsp;
 
+☑️  Describe any other relevant data sharing policies.
 
+- _An example could be the Genomic Data Sharing Policy._
 
+☑️  Describe any reasons that will limit data sharing.Other sharing limitations:
```
Comment (carriewright11) [inline 04-writing_a_DMS_plan.Rmd:139]:
```suggestion

☑️ Describe any reasons that will limit data sharing. Other sharing limitations:
```

**Comment by carriewright11** [comment]
I like it!

File: 04-writing_a_DMS_plan.Rmd (hunk 1 above)
Comment (carriewright11) [reply 04-writing_a_DMS_plan.Rmd:37]: oh nevermind I see it above now :P

File: 04-writing_a_DMS_plan.Rmd [hunk 3]
```diff
@@ -23,42 +18,66 @@ These following are the major sections of the plan:
...
 
-- Describe what data from the project will shared or not shared and justify this decision.
+- _This includes data you think is not considered scientific._
 
-C. Metadata, other relevant data, and associated documentation
+☑️  List any relevant information about how you plan to gather the data.
 
-- List information about metadata.
-- List other possibly relevant data (that you think is not considered scientific).
-- List any relevant information about how you plan to gather the data (ex. study protocols and data collection instruments) that you plan to share to make it easier for others to reproduce your work.
```
Comment (carriewright11) [inline 04-writing_a_DMS_plan.Rmd:43]: hmmm I do think that the info about how you plan to gather the data might still need to be emphasized

Comment (carriewright11) [reply 04-writing_a_DMS_plan.Rmd:43]: nevermind I am blind this morning haha

**Review by carriewright11 — APPROVED** [review]
LGTM!

File: 04-writing_a_DMS_plan.Rmd (hunk 2 above)
Comment (avahoffman) [reply 04-writing_a_DMS_plan.Rmd]: Awesome catch!

**Pull request merged**